### PR TITLE
fix: adjusted theme settings for color fixes, references #1178

### PR DESCRIPTION
**Pull request by ajhollid**
This PR implements some color fixes after refactoring theme

- [x] `theme.palette.error.contrastText` -> `theme.palette.error.main` for bar chart and status label
  - Not sure if this is the correct approach, but the colors appear to be correct.
  
  
image

 
- [x]  Added a `border.light.disabled` and `border.dark.disabled` property to the palette
  - [x] Use `border.light.disabled` and `border.dark.disabled` for `action.disabled`. 
  - Again I'm not sure if this is the correct approach, this will apply a color meant for borders to all disabled states?  

image

@marcelluscaio you know the most about the theme structure so please let me know if there is a better way to resovle these issues!

### CODE REVIEW

File: Client/src/Components/Label/index.jsx [hunk 1]
```diff
@@ -127,11 +127,11 @@ const StatusLabel = ({ status, text, customStyles }) => {
 	const colors = {
 		up: {
 			dotColor: theme.palette.success.main,
-			bgColor: theme.palette.success./* dark */ contrastText,
+			bgColor: theme.palette.success.contrastText /* dark */,
 			borderColor: theme.palette.success.main /* light */,
```
Comment (marcelluscaio) [inline Client/src/Components/Label/index.jsx:131]: I had left the light and dark commented just so we know what was there before, I think we can delete these comments (not necessarily on this pr)

File: Client/src/Utils/Theme/darkTheme.js [hunk 2]
```diff
@@ -23,6 +23,9 @@ const {
 } = colors;
 
 const palette = {
+	action: {
+		disabled: border.dark.disabled,
+	},
```
Comment (marcelluscaio) [inline Client/src/Utils/Theme/darkTheme.js:28]: Is this action.disabled color being called anywhere? I didnt find it

**Review by marcelluscaio — COMMENTED** [review]
Great, Alex! Totally agree with your fixes. 
I just had one question regarding the disabled entry you added

File: Client/src/Utils/Theme/darkTheme.js (hunk 2 above)
Comment (ajhollid) [reply Client/src/Utils/Theme/darkTheme.js:28]: > Is this action.disabled color being called anywhere? I didnt find it

Possible I was playing around and forgot to remove it 🤔 I'll double check it as soon as I get a chance!

Comment (ajhollid) [reply Client/src/Utils/Theme/darkTheme.js:28]: Just had a look, this is required for the disabled borders for inputs

**Pull request merged**